README.rst

```diff
@@ -419,8 +495,16 @@ If a warning doesn't contain a path, ``"cq_default_path"`` from the `configurati
 If not configured, ``.gitlab-ci.yml`` will be used as a fallback path.
 
 You can customize the description with ``"cq_description_template"``, see `configuration file to pass options`_.
-Its value should be a template for Python's |string.Template|_. The template should contain ``$description`` and has
-access to all environment variables, e.g. ``$HOME``.
+Its value should be a template for Python's |string.Template|_. The template has access to all environment variables,
+e.g. ``$HOME``, and other variables that depend on the checker type:
+
+Polyspace
+  Any field of a Polyspace defect can be included by using the corresponding
+  `column title <Exporting Polyspace Results>`_ in lowercase as the variable name.
```

Comment:
I vaguely remember that [Implicit Hyperlink Targets for Section Titles](https://gist.github.com/jwodder/5a9264db8c3c81dad4c01e082515a9f1#id13) are not supported on GitHub. We should use an alternative method to reference a title internally.

Reply:
Fixed the link in 25e465cdb4e2d2b7f0c6db0d50ad1241b70df5ba